curriculum/challenges/english/03-front-end-libraries/jquery/target-html-elements-with-selectors-using-jquery.english.md

```diff
@@ -32,8 +32,7 @@ tests:
   - text: You should only use jQuery to add these classes to the element.
     testString: 'assert(!code.match(/class.*animated/g));'
   - text: Your jQuery code should be within the <code>$(document).ready();</code> function.
-    testString: assert(code.match(/\$\(document\)\.ready\(function.*(\s|\n)*.*button.*.addClass.*\);/g));
-
+    testString: assert(code.match(/\$\(document\)\.ready\(function\s*\(\)\s*{\n\s*\$.*button.*.addClass.*/g));
```

Comment:
```suggestion
    testString: assert(code.match(/\$\(document\)\.ready\(function\s*\(\)\s*{\s*\$\(\s*('|")button\1\s*\)\s*\.\s*addClass[\s\S]*\)/g));
```

Reply:
@RandellDawson and I were thinking we should go back to just using regex. The thinking is that users would have to have the correct code in order to pass even though their code may be in the correct area.

Reply:
@Sky020 If you are ok with @moT01's suggestion above, I am fine with it, however, we could go with a shorter version by just removing the spaces:
```js
assert(code.replace(/\s/g, '').match(/\$\(document\)\.ready\(function\(\)\{\$/g));
```
If you are fine with this, feel free to either commit @moT01's or my suggestion above. Then I can approve/merge this PR.  Thanks.